app/server/app.py

```diff
@@ -2,27 +2,34 @@
 from fastapi.middleware.cors import CORSMiddleware
 from .auth.jwt_bearer import JWTBearer
 from .routes.user import router as UserRouter
-from .routes.admin import router as AdminRouter
 
 app = FastAPI()
 
 origins = [
```

Comment:
dont put url. put ["*"]

Reply:
@PranavBakre Change and push kthnx
Me afk